### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by Grezor** [opcodesio/log-viewer#3] [Ideas]
Hello, 
I wanted to tell you that your project is very clean.
Why not create a dark mode for your application?

**Comment by arukompas** [comment]
I can see it's the most requested (rightfully so, haha) feature, so I'm bumping it up on my priorities list 😄

**Reply by Grezor** [reply]
It's not necessarily a priority, I would love to help you contribute to your project :)

**Comment by lloricode** [comment]
+1

**Comment by arukompas** [comment]
🚀  Dark Mode has been released! Just update to the latest version ⏫
